x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_style_property.js

```diff
@@ -52,6 +53,10 @@ export class DynamicStyleProperty extends AbstractStyleProperty {
     const fieldName = this.getFieldName();
     const rangeFieldMetaFromLocalFeatures = styleMeta.getRangeFieldMetaDescriptor(fieldName);
 
+    if (!this.isFieldMetaEnabled()) {
```

Comment:
Fixes a bug introduced in https://github.com/elastic/kibana/pull/58766 where disabling style meta would still use style meta range, instead of range from local features.

Reply:
🙇